### PR TITLE
Fix/9816 tabbar items low contrast

### PR DESCRIPTION
**Pull request by naveeddotio**
## Description
- Changing current assets 
- Adding tint to unselected tab bar items

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9816

## Screenshots
Low Contrast:
MicrosoftTeams-image (2)
High Contrast:
Screenshot 2022-05-24 at 17 46 15

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/RootCoordinator.swift [hunk 1]
```diff
@@ -277,6 +277,7 @@ class RootCoordinator: NSObject, RequiresAppDependencies, UITabBarControllerDele
 		diaryCoordinator.viewController.tabBarItem = diaryTabBarItem
 
 		tabBarController.tabBar.tintColor = .enaColor(for: .tint)
+		tabBarController.tabBar.unselectedItemTintColor = .enaColor(for: .textPrimary2)
```
Comment (Ein-Tim) [inline src/xcode/ENA/ENA/Source/RootCoordinator.swift:280]: Does this revert the change done in https://github.com/corona-warn-app/cwa-app-ios/pull/3144 (translucent tab bar to match iOS native apps)?

Comment (nickguendling) [inline src/xcode/ENA/ENA/Source/RootCoordinator.swift:280]: I think you've used two incompatible approaches here. You could either add more images (as you did) and set the rendering mode to original, or keep the template images and add high contrast colors in the color asset catalogue and set both tint colors as you did here. I think extending the colors in the asset catalogue would be preferable as it is more flexible (and it would add support for high contrast mode in other places where we use the colors)

**Review by nickguendling — CHANGES_REQUESTED** [review]

**Review by nickguendling — APPROVED** [review]

**Review by ArturFriesen — APPROVED** [review]

**Review by KaiTeuber — APPROVED** [review]
